src/scriv/literals.py

```diff
@@ -98,3 +112,20 @@ def find_toml_value(data: MutableMapping[str, Any], name: str) -> Optional[str]:
     if isinstance(current_object, str):
         return current_object
     return None
+
+def find_yaml_value(data: MutableMapping[str, Any], name: str) -> Optional[str]:
```

Comment:
This function is exactly the same as `find_toml_value`, so we should rename that function instead, and use it for both.  Maybe `find_nested_value`?